projects/uitdatabank/docs/entry-api/places/create.md

```diff
@@ -1,12 +1,316 @@
+
+### mainLanguage
+
+This property indicates the language that the place data is originally entered in. Every translatable property on the place, like `name` or `description`, will need to have at least a value in this language.
+
+Possible values are `nl`, `fr`, `en` or `de`.
+
+Example for a place entered originally in French:
+
+```json
+{
+  "mainLanguage": "fr"
+}
+```
+
+### name
+
+This property contains the human-readable name of the place, shown in online calendars and offline publications to readers.
+
+Because it is translatable, it is an object with language keys as properties for the multiple possible values. At least a value for the language provided as `mainLanguage` is required, but values in other allowed language keys (`nl`, `fr`, `de`, `en`) are also allowed.
+
+Example for a place entered originally in Dutch:
+
+```json
+{
+  "mainLanguage": "nl",
+  "name": {
+    "nl": "Mijn voorbeeld locatie"
+  }
+}
+```
+
+Example for a place entered originally in English, with a Dutch translation:
+
+```json
+{
+  "mainLanguage": "en",
+  "name": {
+    "en": "My example place",
+    "nl": "Mijn voorbeeld locatie"
+  }
+}
+```
+
+While it is possible to immediately provide translations when creating a new place, you can also add them later by [updating the place](update.md).
+
+### address
+
+The address the place is located at. Localized because some parts like the municipality or street address can be different in Dutch, French, German and/or English. The address requires at least one locale, specifically the one defined in the mainLanguage of the place.
+
+<!-- theme: warning -->
+
+> Only add a localized version if it's an official variant!
+
+```json
+{
+   "mainLanguage":"nl",
+   "name":{
+      "name":"Basiliek van Koekelberg"
+   },
+   "address":{
+      "nl":{
+         "addressCountry":"BE",
+         "addressLocality":"Ganshoren",
+         "postalCode":"1083",
+         "streetAddress":"Basiliekvoorplein 1"
+      },
+      "fr":{
+         "addressCountry":"BE",
+         "addressLocality":"Ganshoren",
+         "postalCode":"1083",
+         "streetAddress":"Parvis de la Basilique 1"
+      }
+   }
+}
+```
+
+### terms
+
+This property contains a list of [taxonomy terms](../../taxonomy-api/terms.md) that categorize the place in one of various types. Specific partners may also add accessibility facilities as terms.
+
+The possible taxonomy terms can be browsed via the [`GET /terms`](/reference/taxonomy.json/paths/~1terms/get) endpoint on the Taxonomy API.
+
+To create a place, at least one term of the domain `eventtype` and a `scope` that contains `places` is required. For example the following term that categorizes a place as a museum:
```

Comment:
This one is a bit confusing, while the `API` apparantly allows multiple `termIds` only 1 is saved.
Reading this I would conclude it would be possible to create a place that could be both a museum and a library.

Reply:
I made the copy clearer in commit `c8f73d7`